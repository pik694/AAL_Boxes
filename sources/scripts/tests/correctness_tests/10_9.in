10
34 27 91
32 32 74
56 39 12
82 81 52
85 97 2
29 30 51
29 57 7
66 46 66
57 9 94
65 94 3

