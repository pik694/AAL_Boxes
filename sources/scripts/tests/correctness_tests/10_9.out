6 8 4 9 2 7 3 5 1 0 
870265->587740
